### DISCUSSION TITLE
Use bruteforce tool api

### DISCUSSION THREAD

**Discussion by Darkhaxxor005** [Alygnt/Clifty#36] [Ideas]
Sometimes victims put invalid credentials in the page to check,  that is why a bruteforce tool api for facebook can check for the given credentials of victim if it is valid it will send it to attacker otherwise a incorrect email or password will show on the phishing site,  it will make the site more trusted and eligible.

**Comment by Alygnt** [comment]
I like your idea, I will try to implement it in the next update

**Reply by Darkhaxxor005** [reply]
I have email module that will send mail to attacker with information

**Reply by Alygnt** [reply]
Can you just mail it to mralygnt@gmail.com
